tools/clang/lib/SPIRV/ExecutionModel.h

```diff
@@ -1,62 +0,0 @@
-//===------- ExecutionModel.h - get/set SPV Execution Model -----*- C++ -*-===//
```

Comment:
Happy to see that the class is gone. Especially the `static` data that was included in it.